cmd/describe/cluster/cmd.go

```diff
@@ -335,6 +286,74 @@ func run(cmd *cobra.Command, argv []string) {
 	fmt.Println()
 }
 
+func controlPlaneConfig(cluster *cmv1.Cluster) string {
+	if cluster.Hypershift().Enabled() {
+		return "Red Hat hosted"
+	}
+	return "Customer hosted"
+}
+
+func clusterInfraConfig(cluster *cmv1.Cluster, clusterKey string, r *rosa.Runtime) string {
+	var infraConfig string
+	if cluster.Hypershift().Enabled() {
+		infraConfig = fmt.Sprintf(""+
+			"Node Pools:\n"+
+			" - Compute:                 %d\n",
+			cluster.Nodes().Compute(),
```

Comment:
I think we still have to do everything below just don't display the master and infra. we need autoscaling and replicas same as regular rosa sts

Reply:
CS does not persist the nodepool info in the database; there is no reconciliation on NodePool CR to determine the replica size as well. For the first iteration, we can only show the number of worker nodes (as it is stored in the db). The second iteration will add more info to this field.

Reply:
we need to persist the nodepool info, otherwise this becomes a problem for tracking quota consumption in AMS and any cluster reshape will drift the cluster authorizations.

Reply:
Right, we have two cards to address this: one to persist nodepool in db (that I am working on); the other is to consume NodePool status to fetch the autoscaling info. For the first iteration, we just show the node count.